conf/nonclangable.conf

```diff
@@ -1,3 +1,20 @@
+#| clang-15: error: unable to execute command: Segmentation fault (core dumped)
+#| clang-15: error: linker command failed due to signal (use -v to see invocation)
+TOOLCHAIN:pn-pciutils:riscv64 = "gcc"
+
+#| erl_bits.c:(.text+0xc2a): undefined reference to `__extendhfsf2'
+#| erl_bits.c:(.text+0x1bfa): undefined reference to `__truncsfhf2'
+#| clang-15: error: linker command failed with exit code 1 (use -v to see invocation)
+# both riscv64 and x86-64
+TOOLCHAIN:pn-erlang = "gcc"
+
+# run.do_uboot_mkimage.1228: 185: riscv64-yadro-linux-llvm-objcopy: not found
+# FileNotFoundError: [Errno 2] No such file or directory: 'x86_64-yadro-linux-llvm-objcopy'
+# both riscv64 and x86-64
+OBJCOPY:pn-linux-yocto:toolchain-clang = "${HOST_PREFIX}objcopy"
```

Comment:
There already is similar setting for `mips` perhaps we need to remove that also if we set it globally like this.